deploy/180_check_SUSDMetaPoolTokens.ts

```diff
@@ -0,0 +1,38 @@
+import { DeployFunction } from "hardhat-deploy/types"
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { isTestNetwork } from "../utils/network"
+import { BigNumber } from "ethers"
+
+const USD_TOKENS_ARGS: { [token: string]: any[] } = {
+  SUSD: ["Synth sUSD", "sUSD", "18"],
+}
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, getChainId } = hre
+  const { deploy, execute } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  for (const token in USD_TOKENS_ARGS) {
+    await deploy(token, {
+      from: deployer,
+      log: true,
+      contract: "GenericERC20",
+      args: USD_TOKENS_ARGS[token],
+      skipIfAlreadyDeployed: true,
```

Comment:
because we have the `skipIfAlreadyDeployed`, should this also redeploy the usdv2 pool LP token?

Reply:
The expected usage is to call this script via the `tag` option so that it also runs all other scripts under `dependencies`. So it will redeploy usdv2 pool and the lp token.